eth/db/chain.py

```diff
@@ -143,6 +143,12 @@ def get_transaction_by_index(
     def get_transaction_index(self, transaction_hash: Hash32) -> Tuple[BlockNumber, int]:
         raise NotImplementedError("ChainDB classes must implement this method")
 
+    @abstractmethod
```

Comment:
nitpick: would be nice to have this directly next to `get_receipts`.

Reply:
Done :+1: